From cfd39bb5fb7e44ab18cf0b85f63a541d34dbe469 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 9 Aug 2022 09:59:41 -0700
Subject: [PATCH] monkey: Define _GNU_SOURCE for memmem API check

This define is necessary to get this API on glibc based systems

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 lib/monkey/mk_core/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/monkey/mk_core/CMakeLists.txt b/lib/monkey/mk_core/CMakeLists.txt
index a1a3210..033a9cb 100644
--- a/lib/monkey/mk_core/CMakeLists.txt
+++ b/lib/monkey/mk_core/CMakeLists.txt
@@ -65,6 +65,7 @@ set(src "${src}"
   )
 
 check_c_source_compiles("
+  #define _GNU_SOURCE
   #include <string.h>
   int main() {
      char  haystack[] = \"1234\";
